src/components/backend-ai-data-view.ts

```diff
@@ -229,18 +229,22 @@ export default class BackendAIData extends BackendAIPage {
           --mdc-select-dropdown-icon-color: var(--general-textfield-selected-color);
           --mdc-select-hover-line-color: var(--general-textfield-selected-color);
           --mdc-list-vertical-padding: 5px;
+          /* Need to be set when fixedMenuPosition attribute is enabled */
+          --mdc-menu-max-width: 340px;
+          --mdc-menu-min-width: 340px;
         }
 
-        mwc-select.full-width {
+        mwc-select.full-width.fixed-position {
           width: 100%;
+          /* Need to be set when fixedMenuPosition attribute is enabled */
+          --mdc-menu-max-width: 345px;
+          --mdc-menu-min-width: 345px;
```

Comment:
Please reduce the width of type like project, usage mode, and permission.
<img width="449" alt="image" src="https://user-images.githubusercontent.com/28584164/178411985-1e0dcaf8-46d2-4442-aa80-5ec00c2548c7.png">

Reply:
fixed it. 
```css
        mwc-select {
          width: 50%;
          margin-bottom: 10px;
          --mdc-theme-primary: var(--general-textfield-selected-color);
          --mdc-select-fill-color: transparent;
          --mdc-select-label-ink-color: rgba(0, 0, 0, 0.75);
          --mdc-select-dropdown-icon-color: var(--general-textfield-selected-color);
          --mdc-select-hover-line-color: var(--general-textfield-selected-color);
          --mdc-list-vertical-padding: 5px;
          /* Need to be set when fixedMenuPosition attribute is enabled */
          --mdc-menu-max-width: 340px;
          --mdc-menu-min-width: 170px;
        }

        mwc-select.full-width.fixed-position {
          width: 100%;
          /* Need to be set when fixedMenuPosition attribute is enabled */
          --mdc-menu-max-width: 340px;
          --mdc-menu-min-width: 340px;
        }
```